tests/drivers/gpio/gpio_api_1pin/boards/native_posix.overlay

```diff
@@ -5,5 +5,31 @@
  */
 
 &gpio0 {
-	ngpios = <2>;
+	ngpios = <3>;
+};
+
+/ {
+	gpio-keys0 {
+		compatible = "zephyr,gpio-keys";
+		debounce-interval-ms = <30>;
+
+		voldown_button: button_0 {
+			gpios = <&gpio0 0 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			zephyr,code = <10>;
+		};
+		volup_button: button_1 {
+			gpios = <&gpio0 1 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			zephyr,code = <11>;
```

Comment:
Guess we may want to import a chunk of `include/uapi/linux/input-event-codes.h` down the road. (@cfriedt)